spyder/plugins/editor/plugin.py

```diff
@@ -1060,7 +1060,7 @@ def get_plugin_actions(self):
             self.new_action,
             menu_id=ApplicationMenus.File,
             section=FileMenuSections.New,
-            before_section=FileMenuSections.Restart,
+            before_section=FileMenuSections.Open,
```

Comment:
Checking, a way to work around the issue with the actions sections order in the `File` menu is to change the calling order for adding the actions sections using the `before_section` kwarg to the inverse. So currently the order here is:

```
# New Section
 ...
# Open section
...
# Save section
...
# Print
...
# Close
...
# Navigation
```

To workaround the issue, the calling order should be:

```
# Navigation
...
# Close
...
# Print
...
# Save section
...
# Open section
...
# New Section
...
```

Reply:
You are right if the adding order is changed it works [8b67fe4](https://github.com/spyder-ide/spyder/pull/20714/commits/8b67fe4db8dab0e47d4093af14b331531486b64f)

Reply:
![image](https://user-images.githubusercontent.com/10239480/230618267-b5fcd4f6-84f7-4930-acb2-b0e35feead48.png)